cypress-tests/package.json

```diff
@@ -14,7 +14,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "cypress": "^13.10.0",
+    "cypress": "^13.12.0",
```

Comment:
This change can be reverted as my PR takes care of it.

Reply:
Sure, if so then the change will anyways will be overwritten.